packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -1,21 +1,39 @@
-import { IDesignSystem } from "../design-system";
+import designSystemDefaults, { IDesignSystem } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
 import { applyLocalizedProperty, toPx } from "@microsoft/fast-jss-utilities";
 import { typeRamp } from "../utilities/typography";
 import { IToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { get } from "lodash";
 import * as Chroma from "chroma-js";
 import { Direction } from "@microsoft/fast-application-utilities";
 
-function setFloatAndClear(clearFloat: boolean = true): ICSSRules<IDesignSystem> {
+function setLocalizedFloat(): ICSSRules<IDesignSystem> {
     return {
-        float: (config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right",
-        clear: clearFloat ? ((config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right") : null
+        float: (config: IDesignSystem): string => {

```

Comment:
what if I want to float something right in LTR? Maybe just a generic "applyLocalizedValue" would work for both of these - does that exist yet?:

```
float: applyLocalizedValue("left", "right"),
clear: applyLocalizedValue("left", "right")
```

Reply:
yeah - this makes way more sense. At this point, the function really doesn't provide a lot of value.

Reply:
@nicholasrice updated with applyLocalizedProperty from jss-utils package.